pkg/pillar/cmd/downloader/downloader.go

```diff
@@ -1518,11 +1518,52 @@ func rktFetch(url string, localConfigDir string, pullPolicy string) (string, err
 		return "", errors.New(errMsg)
 	}
 
+	err = rktImageExport(imageID, imageUUID, locFilename)
+	if err != nil {
+		return imageID, err
+	}
```

Comment:
If we can't export - do not return ImageID?? Whats the point of returning ImageID? Just log it..